### PR TITLE
chore(docs): enhance docs showcase visibility & update icons in docs tab

### PR DESCRIPTION
**Pull request by dannysheridan**
## Changes Made
- edit icons for ~40 docs pages
- move docs showcase "higher" in the information architecture to make it more prevalent 

## Testing
- fern check passes
- need to review a pr preview link and check out the icons (wasn't tested locally)

### CODE REVIEW

File: fern/docs.yml [hunk 1]
```diff
@@ -528,6 +528,7 @@ navigation:
             contents:
               - page: Intercom
                 path: ./pages/docs/integrations/support/intercom.mdx
+                icon: fa-brands fa-intercom
           - page: Postman
```
Comment (chdeskur) [inline fern/docs.yml:532]: thoughts on using an astronaut icon here?

File: fern/docs.yml [hunk 2]
```diff
@@ -467,35 +467,35 @@ navigation:
...
           - page: Changelog
             path: ./pages/docs/writing-content/changelog.mdx
-            icon: fa-regular fa-clock
+            icon: fa-regular fa-clock-rotate-left
 
       - section: API References
         contents:
           - page: Generate API Reference
-            icon: fa-regular fa-gear-code
+            icon: fa-regular fa-gear
```
Comment (chdeskur) [inline fern/docs.yml:475]: we use gear already, thoughts on `square-plus`? or `folder-plus`?

**Review by chdeskur — APPROVED** [review]

File: fern/docs.yml (hunk 1 above)
Comment (dannysheridan) [reply fern/docs.yml:532]: assuming this is for postman

File: fern/docs.yml (hunk 2 above)
Comment (dannysheridan) [reply fern/docs.yml:475]: I want to emphasize that it's automatically generated. going to try `fa-solid fa-wand-magic-sparkles`

**Pull request merged**